app/i18n/locale_en.json

```diff
@@ -13,7 +13,7 @@
     "tab.national": "National",
     "tab.national-view": "National View",
     "tab.school-view": "Schools",
-    "tab.school": "schools",
+    "tab.school": "Schools",
```

Comment:
I think it's ok -- we can `style: lowercase` if we need it lowered, which has less edge cases for CSS to mess up than `capitalize`.